scripts/fixate-requirements.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+
+import os
+import os.path
+import sys
+from distutils.version import StrictVersion
+
+
+cwd = os.path.abspath(os.getcwd())
+FIXED_REQF = os.path.join(cwd, '../fixed-requirements.txt')
+LOCAL_REQF = os.path.join(cwd, 'requirements.txt')
+GET_PIP = "    curl https://bootstrap.pypa.io/get-pip.py | python"
+
+try:
+    import pip
+    from pip.req import parse_requirements
+except ImportError:
+    print "Download pip:\n", GET_PIP
+    sys.exit(1)
+
+
+def check_pip_version():
+    if StrictVersion(pip.__version__) < StrictVersion('7.0.0'):
+        print "Upgrade pip, your version `{}' "\
```

Comment:
Yeah sure. Got it. Ruby had known distinctions between ' and ", that's why
I automatically chosen quotes without thinking.

On Tue, Jun 23, 2015, 07:58 Lakshmi Kannan <notifications@github.com> wrote:

> In scripts/fixate-requirements.py
> <https://github.com/StackStorm/st2/pull/1647#discussion_r33009484>:
>
> > +cwd = os.path.abspath(os.getcwd())
> > +FIXED_REQF = os.path.join(cwd, '../fixed-requirements.txt')
> > +LOCAL_REQF = os.path.join(cwd, 'requirements.txt')
> > +GET_PIP = "    curl https://bootstrap.pypa.io/get-pip.py
>
>  | python"
> > +
> > +try:
> > +    import pip
> > +    from pip.req import parse_requirements
> > +except ImportError:
> > +    print "Download pip:\n", GET_PIP
> > +    sys.exit(1)
> > +
> > +
> > +def check_pip_version():
> > +    if StrictVersion(pip.__version__) < StrictVersion('7.0.0'):
> > +        print "Upgrade pip, your version `{}' "\
>
> Nit: Can you use print as methods? Also, we use '' for strings. PEP-8.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/StackStorm/st2/pull/1647/files#r33009484>.
>